index.html

```diff
@@ -571,70 +571,98 @@ <h3>External Binary Content</h3>
-          or <code>PUT</code>, that the Fedora server should not consider the request entity to be the <a>LDP-NR</a>'s
-          content, but that a <code>Content-Type</code> value will signal a name or address at which the content might
-          be retrieved. The <code>URL</code> ([[RFC2017]]
-          <a href="https://tools.ietf.org/html/rfc2017#section-3">section 3</a>) and <code>local-file</code>
-          ([[RFC2046]] <a href="https://tools.ietf.org/html/rfc2046#section-5.2.3.4">section 5.2.3.4</a>) values of the
-          <code>access-type</code> parameter motivate this specification, but a Fedora server may support any
-          <code>access-type</code> parameters per the requirements for advertisement and rejection specified here.
+          describes the use of a <code>rel="http://fedora.info/definitions/fcrepo#ExternalContent"</code> link in a
+          <code>Link</code> header to signal, on <code>POST</code> or <code>PUT</code>, that the Fedora server should
+          not consider the request entity to be the <a>LDP-NR</a>'s content, but instead that the target of the
+          link gives the URI from which the content might be retrieved. The <code>http(s):</code> [[RFC3986]] and
+          <code>file:</code> [[RFC8089]] URI schemes motivate this specification, but a Fedora server may support
+          other URI schemes for addressing external content per the requirements for advertisement and rejection
+          specified here. The <code>handling</code> attribute is introduced to specify the handling of the external
+          content: either copying, redirecting, or proxying.
         </blockquote>
-        <p id='message-external-body-accept-post'>
-          Fedora servers SHOULD support the creation of <a>LDP-NR</a>s with
-          <code>Content-Type: message/external-body</code> and <code>access-type</code> parameter value
-          <code>URL</code>.
-        </p>
-        <p id='message-external-body-supported-type'>
-          Fedora servers MUST advertise support in the <code>Accept-Post</code> response header for each supported
-          <code>access-type</code> parameter value of <code>Content-Type: message/external-body</code>.
-        </p>
-        <p id='message-external-body-unsupported-type'>
-          Fedora servers receiving requests that would create or update an <a>LDP-NR</a> with
-          <code>Content-Type: message/external-body</code> and an unsupported <code>access-type</code> parameter value
-          MUST respond with 415 (Unsupported Media Type). In the case that a Fedora server does not support external
-          <a>LDP-NR</a> content,
-          all <code>message/external-body</code> messages must be rejected with 415 (Unsupported Media Type).
+        <div class="example">
+<pre>Link: <i>&lt;URL&gt;</i>;
+      rel="http://fedora.info/definitions/fcrepo#ExternalContent";
+      handling="<i>&lt;copy|redirect|proxy&gt;</i>";
+      type="<i>&lt;media-type&gt;</i>"
```

Comment:
Would it be clearer to use actual examples here instead of the variables, e.g.,

```
Link: <http://example.org/some/content>; 
      rel="http://fedora.info/definitions/fcrepo#ExternalContent"; 
      handling="proxy"; 
      type="image/tiff"
```

Reply:
I agree, will fix